docs/content/faq.md

```diff
@@ -184,23 +184,25 @@ The following configuration works nicely:
     # This assumes you're having `environment: ONE_DIR=1` in the env-mailserver,
     # with a consolidated config in `/var/mail-state`
     #
+    # '> /dev/null' to send error notifications from 'stderr' to 'postmaster@example.com'
```

Comment:
It's technically meant to omit `stdout` to avoid reports that are unnecessary, thus only `stderr` would trigger reports.